observability/README.md

```diff
@@ -8,7 +8,42 @@ In this quickstart you will:
 - Instrument an application for tracing and then deploy it.
 - Troubleshoot a performance issue.
 
-## Prerequisites
+## Configure self hosted mode
+For self hosted mode, on running `dapr init`:
```

Comment:
This is little unclear. Does this require `dapr init` first?

Reply:
I reworded.